cmd/talosctl/cmd/talos/config.go

```diff
@@ -175,8 +175,40 @@ func openConfigAndContext(context string) (*clientconfig.Config, error) {
 	return c, nil
 }
 
+// configMergeCmd represents the config merge command.
+var configMergeCmd = &cobra.Command{
+	Use:    "merge <from>",
+	Short:  "Merge in additional contexts from another Talos config",
+	Long:   ``,
```

Comment:
we could have used long to say that on name conflict context is renamed to help users trust this command